lib/src/ignore.dart

```diff
@@ -150,16 +150,18 @@ class Ignore {
         path.endsWith('/') ? path.substring(0, path.length - 1) : path;
     return listFiles(
       beneath: pathWithoutSlash,
-      includeDirs: true,
+      includeDirs: true, // because we listing below pathWithoutSlash
```

Comment:
Broken sentence maybe?:
```suggestion
      includeDirs: true, // because we are listing below pathWithoutSlash
```